conformity/fields/email.py

```diff
@@ -49,8 +49,9 @@ def __init__(self, message=None, code=None, whitelist=None):
         :param whitelist: If specified, an invalid domain part will be permitted if it is in this list
         :type whitelist: iterable
         """
+        super(UnicodeString, self).__init__()
```

Comment:
@nickwilliams-eventbrite This doesn't solves the issue as the constructor is never being called. I added another test that fails (the one that checks if I add repeated whitelist items they get removed due to line 54's `set(whitelist)`), but it is as simple as adding a `exit(1)` inside the constructor and all tests but one will still pass. 
Current constructor code is never called due to the `@attr.s` class decorator.

I'll work on a proper solution, taking into account we wish to explicitly have non-repeated items/a set, as soon as I can.

Reply:
Thanks for the explanation, indeed that was the issue.